### PR TITLE
update the main doc and add a developer doc

### PR DESCRIPTION
**Pull request by wenbingl**

### CODE REVIEW

File: tutorials/bert_sentiment.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import onnx
```
Comment (natke) [inline tutorials/bert_sentiment.py:1]: Does this work end to end?

File: README.md [hunk 2]
```diff
@@ -212,7 +74,7 @@ The PyTorch and TensorFlow converters support custom operator generation if the
 2. [CustomOp conversion by tf2onnx](https://github.com/microsoft/onnxruntime-extensions/blob/main/tutorials/tf2onnx_custom_ops_tutorial.ipynb)
 
 
-## Contribute a new operator to onnxruntime-extensions
+## Add a new custom operator to onnxruntime-extensions
 
 Welcome to contribute the customop C++ implementation directly in this repository, which will widely benefit other users. Besides C++, if you want to quickly verify the ONNX model with some custom operators with Python language, PyOp will help with that
```
Comment (natke) [inline README.md:79]:
```suggestion
You can contribute customop C++ implementation directly in this repository if they have general applicability to other users. In addition, if you want to quickly verify the ONNX model with Python, you can wrap the custom operator with PyOp.
```

**Review by natke — COMMENTED** [review]
We should update the CI matrix to use ONNX Runtime 1.13

File: tutorials/bert_sentiment.py (hunk 1 above)
Comment (wenbingl) [reply tutorials/bert_sentiment.py:1]: Yes, I verified it locally. It's your contribution but I don't know how I can put a history there!

File: docs/development.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Build and Development
+
+This project supports Python and can be built from source easily, or a simple cmake build without Python dependency.
+## Python package
+The package contains all custom operators and some Python scripts to manipulate the ONNX models.
+- Install Visual Studio with C++ development tools on Windows, or gcc(>8.0) for Linux or xcode for macOS, and cmake on the unix-like platform. (**hints**: in Windows platform, if cmake bundled in Visual Studio was used, please specify the set _VSDEVCMD=%ProgramFiles(x86)%\Microsoft Visual Studio\<VERSION_YEAR>\<Edition>\Common7\Tools\VsDevCmd.bat_)
+- If running on Windows, ensure that long file names are enabled, both for the [operating system](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd) and for git: `git config --system core.longpaths true`
```
Comment (sayanshaw24) [inline docs/development.md:7]: When enabling long paths for the operating system, this linked site was useful but I ran into some issues and the directions on this other site helped: https://learn.microsoft.com/en-us/answers/questions/730467/long-paths-not-working-in-windows-2019.html

We could add this as well for user reference.

**Review by sayanshaw24 — APPROVED** [review]
Looks great! Thanks for adding this.

File: docs/development.md (hunk 3 above)
Comment (wenbingl) [reply docs/development.md:7]: Feel free to create a PR to enhance the section in this document. I can only share what I knew here.

**Pull request merged**